tox.ini

```diff
@@ -12,6 +12,7 @@ deps =
     pytest==3.0.7
     pytest-django==3.1.2
     django-compressor==2.1.1
+    jinja2==2.9.6
```

Comment:
Currently it isn't very intuitive to test multiple versions locally. Could we revamp the `tox.ini` to make this easier? I wrote one for [django-safedelete](https://github.com/makinacorpus/django-safedelete/blob/master/tox.ini) as well.